torchvision/transforms/v2/functional/_meta.py

```diff
@@ -176,6 +176,33 @@ def _xyxy_to_cxcywh(xyxy: torch.Tensor, inplace: bool) -> torch.Tensor:
     return xyxy
 
 
+def _cxcywhr_to_xywhr(cxcywhr: torch.Tensor, inplace: bool) -> torch.Tensor:
+    if not inplace:
+        cxcywhr = cxcywhr.clone()
+
+    half_wh = cxcywhr[..., 2:-1].div(-2, rounding_mode=None if cxcywhr.is_floating_point() else "floor").abs_()
+    r_rad = cxcywhr[..., 4].mul(torch.pi).div(180.0)
+    # (cx - width / 2 * cos - height / 2 * sin) = x1
+    cxcywhr[..., 0].sub_(half_wh[..., 0].mul(r_rad.cos()).to(cxcywhr.dtype)).sub_(half_wh[..., 1].mul(r_rad.sin()).to(cxcywhr.dtype))
+    # (cy + width / 2 * sin - height / 2 * cos) = y1
+    cxcywhr[..., 1].add_(half_wh[..., 0].mul(r_rad.sin()).to(cxcywhr.dtype)).sub_(half_wh[..., 1].mul(r_rad.cos()).to(cxcywhr.dtype))
+
+    return cxcywhr
+
+
+def _xywhr_to_cxcywhr(xywhr: torch.Tensor, inplace: bool) -> torch.Tensor:
+    if not inplace:
+        xywhr = xywhr.clone()
+
+    half_wh = xywhr[..., 2:-1].div(-2, rounding_mode=None if xywhr.is_floating_point() else "floor").abs_()
+    r_rad = xywhr[..., 4].mul(torch.pi).div(180.0)
+    # (x1 + width / 2 * cos + height / 2 * sin) = cx
+    xywhr[..., 0].add_(half_wh[..., 0].mul(r_rad.cos()).to(xywhr.dtype)).add_(half_wh[..., 1].mul(r_rad.sin()).to(xywhr.dtype))
+    # (y1 - width / 2 * sin + height / 2 * cos) = cy 
+    xywhr[..., 1].sub_(half_wh[..., 0].mul(r_rad.sin()).to(xywhr.dtype)).add_(half_wh[..., 1].mul(r_rad.cos()).to(xywhr.dtype))
+
+    return xywhr
+
```

Comment:
I think we should try to implement conversions to/from XYXYXYXY as well here. I'm assuming you didn't implement it because there is no way to do it inplace? I think that's OK, we can simply document that the `inplace`  parameter is ignored (and assumed to be False) when dealing with XYXYXYXY formats.